xml/System.Windows.Xps.Packaging/IXpsFixedDocumentWriter.xml

```diff
@@ -1,241 +1,246 @@
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_Wpf/XpsCreate/CSharp/XpsCreate.cs" id="Snippetxpscreateaddpkgcontent":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_Wpf/XpsCreate/visualbasic/xpscreate.vb" id="Snippetxpscreateaddpkgcontent":::  
+  
+ ]]></format>
+    </remarks>
+  </Docs>
+  <Members>
+    <Member MemberName="AddFixedPage">
+      <MemberSignature Language="C#" Value="public System.Windows.Xps.Packaging.IXpsFixedPageWriter AddFixedPage ();" />
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig newslot virtual instance class System.Windows.Xps.Packaging.IXpsFixedPageWriter AddFixedPage() cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.Windows.Xps.Packaging.IXpsFixedDocumentWriter.AddFixedPage" />
+      <MemberSignature Language="VB.NET" Value="Public Function AddFixedPage () As IXpsFixedPageWriter" />
+      <MemberSignature Language="F#" Value="abstract member AddFixedPage : unit -&gt; System.Windows.Xps.Packaging.IXpsFixedPageWriter" Usage="iXpsFixedDocumentWriter.AddFixedPage " />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; System::Windows::Xps::Packaging::IXpsFixedPageWriter ^ AddFixedPage();" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>ReachFramework</AssemblyName>
+        <AssemblyVersion>3.0.0.0</AssemblyVersion>
+        <AssemblyVersion>4.0.0.0</AssemblyVersion>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Windows.Xps.Packaging.IXpsFixedPageWriter</ReturnType>
+      </ReturnValue>
+      <Parameters />
+      <Docs>
+        <summary>Adds a new <see cref="T:System.Windows.Documents.FixedPage" /> to the <see cref="T:System.Windows.Documents.FixedDocument" />.</summary>
+        <returns>A writer for outputting the new <see cref="T:System.Windows.Documents.FixedPage" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Examples  
+ The following example shows how to use the <xref:System.Windows.Xps.Packaging.IXpsFixedDocumentWriter.AddFixedPage%2A> method.  
+  
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_Wpf/XpsCreate/CPP/XpsCreate.cpp" id="Snippetxpscreateadddoccontent":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_Wpf/XpsCreate/CSharp/XpsCreate.cs" id="Snippetxpscreateadddoccontent":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_Wpf/XpsCreate/visualbasic/xpscreate.vb" id="Snippetxpscreateadddoccontent":::  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.ObjectDisposedException">The FixedDocument has already been disposed</exception>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PanelOrSequenceAlreadyOpen">FixedPage is not completed.</exception>

```

Comment:
```suggestion
        <exception cref="T:System.ObjectDisposedException">The fixed document has already been disposed</exception>
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PanelOrSequenceAlreadyOpen">The fixed page is not completed.</exception>

```

Reply:
FixedDocument and FixedPage are classes.